scilpy/viz/color.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
If my PR #950 is merged first, we could move the this I put in viz.utils here.

Reply:
@EmmaRenauld what were you talking about ?

Reply:
I'm pretty sure I did the move, I'll let you confirm @EmmaRenauld